.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,10 +11,7 @@ Thank you for reporting the problem!
 Please make sure what you are reporting is a bug with reproducible steps. 
 
 **Describe the bug**
-A clear and concise description of what the bug is.
-
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+A clear and concise description of what the bug is, and clear and concise description of what you expected to happen.
```

Comment:
"**Describe the bug**" is more vauge than "Expected behavior" + "Actual behavior". Because different people have different understanding of how to "**Describe the bug**". And the longer explanation is less possible to read carefully.

Reply:
Maybe take a look at https://github.com/apache/kvrocks/blob/unstable/.github/ISSUE_TEMPLATE/bug-report.yml